### PR TITLE
Javascript/javascript1/week2

### PR DESCRIPTION
**Pull request by Yasser2H**
JavaScript - JavaScript1 -  Week2 Homework

### CODE REVIEW

File: javascript/javascript1/week2/script.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// Flight booking fullname function
+console.log("------------------ 1st Task ------------------");
```
Comment (obelitz) [inline javascript/javascript1/week2/script.js:2]: Nice use of console.logs to divide the task.

File: javascript/javascript1/week2/script.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+console.log("2nd Fullname: " + fullname2);
+
+function getFullname(firstName, lastName, useFormalName = false){
+  if (useFormalName) {
+    return ("Lord " + firstName + " " + lastName);
+  } else {
+    return (firstName + " " + lastName);
+  }
+
+}
```
Comment (obelitz) [inline javascript/javascript1/week2/script.js:16]: You would usually place the function first, so it's "created" before you use it.

File: javascript/javascript1/week2/script.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+// Event application
+console.log("------------------ 3rd Task ------------------");
+
+console.log(getEventWeekday(9));
+console.log(getEventWeekday(3));
+
+function getEventWeekday(daysUntilEvent) {
+  const days = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"];
+  let eventDay = days[(new Date().getDay() + daysUntilEvent) % 7];
```
Comment (obelitz) [inline javascript/javascript1/week2/script.js:33]: This doesn't change so it should be assigned as a `const`.

Comment (obelitz) [reply javascript/javascript1/week2/script.js:33]: To make this more readable, I'd assign like this:
```javascript
const today = new Date().getDay();
const eventWeekdayIndex = (today + daysUntilEvent) % 7;
const eventWeekday = days[eventWeekdayIndex];
```

**Review by obelitz — COMMENTED** [review]
Nice - very clean code.

Some of the assignments are missing:
- [Weather Wear](https://github.com/HackYourFuture-CPH/JavaScript/blob/master/javascript1/week2/homework.md#weather-wear)
- [Student Manager](https://github.com/HackYourFuture-CPH/JavaScript/blob/master/javascript1/week2/homework.md#student-manager)

**Comment by Yasser2H** [comment]
Sorry, I forgot to commit the final changes, and also I implemented your suggestions, thank you for your time and efforts :)

**Pull request merged**